salt/modules/ssh.py

```diff
@@ -65,8 +65,8 @@ def _refine_enc(enc):
         return enc
     elif enc in ed25519:
         return "ssh-ed25519"
-    else:
-        raise CommandExecutionError("Incorrect encryption key type '{}'.".format(enc))
+
```

Comment:
looks like @dwoz preferred option 1 from this comment https://github.com/saltstack/salt/issues/59429#issuecomment-794576844

@dwoz would you mind reviewing here.